dashboards-reports/public/components/report_definitions/delivery/delivery.tsx

```diff
@@ -34,19 +34,30 @@ import {
   EuiHorizontalRule,
   EuiSpacer,
   EuiCheckbox,
+  EuiComboBox,
+  EuiFieldText,
+  EuiButton,
 } from '@elastic/eui';
-import { DELIVERY_TYPE_OPTIONS } from './delivery_constants';
+import CSS from 'csstype';
+import { placeholderChannels, testMessageConfirmationMessage } from './delivery_constants';
 import 'react-mde/lib/styles/css/react-mde-all.css';
 import { reportDefinitionParams } from '../create/create_report_definition';
-import { EmailDelivery } from './email';
+import ReactMDE from 'react-mde';
+import { converter } from '../utils';
+
+const styles: CSS.Properties = {
+  maxWidth: '800px',
+};
+
+export let includeDelivery = false;
```

Comment:
why export the variable instead of passing it to utils function call?

Reply:
I can modify to pass to the utils function call but it would still be exported- in the current implementation the input validation all happens at the `Create` and `Edit` level, and since Delivery is optional it is not enough to just check if, for example, configIds is empty. We also need a check on whether Delivery is configured

Reply:
Initially I tried to pass useState hook as props into Delivery but the value of the entire report definition request resets when the setState is called